flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliTableauResultViewTest.java

```diff
@@ -300,11 +300,11 @@ public void testEmptyBatchResult() {
         view.close();
 
         assertThat(terminalOutput.toString()).isEqualTo("Empty set" + System.lineSeparator());
```

Comment:
```suggestion
        assertThat(terminalOutput).hasToString("Empty set" + System.lineSeparator());
```
could be a bit simplified